### PR TITLE
Add Homebridge docs

### PR DESCRIPTION
**Pull request by Zer0x00**

### CODE REVIEW

File: docs/software/home_automation.md [hunk 1]
```diff
@@ -221,4 +222,61 @@ Official website: <https://www.openhab.org/>
...
+    ```sh
+    hb-service restart
+    ```
+
+=== "Update Node.js"
+
+    Use the following command to update the bundled Node.js runtime:
+
+    ```sh
+    # don't use sudo if running from the Homebridge UI Terminal
```
Comment (MichaIng) [inline docs/software/home_automation.md:275]:
```suggestion
    Use the following command to update the bundled Node.js runtime (do not use `sudo` if running from the Homebridge UI Terminal):

    ```sh
```

File: docs/software/home_automation.md [hunk 2]
```diff
@@ -221,4 +222,61 @@ Official website: <https://www.openhab.org/>
...
+=== "Access to the web interface"
+
+    The web interface is accessible via port **8581**:
+
+    - URL: `https://<your.IP>:8581`
+    - Username/Password: Are set at first web interface login
+
+=== "Directories"
+
+    - Install directory: `/usr/local/bin/hb-service`
```
Comment (MichaIng) [inline docs/software/home_automation.md:240]: This is the executable symlink. What comes closest to the install directory is `/opt/homebridge`, which also contains the actual binary.

File: docs/software/home_automation.md [hunk 3]
```diff
@@ -221,4 +222,61 @@ Official website: <https://www.openhab.org/>
...
+    - Config, Log & Backup folder: `/var/lib/homebridge`
+
+=== "View logs"
+
+    ```sh
+    journalctl -u homebridge
+    cat /var/lib/homebridge/homebridge.log
```
Comment (MichaIng) [inline docs/software/home_automation.md:247]: Might be actually an idea to move all logs to journal. Is the `homebridge.log` actually already a duplicate of what is logged to stdout/journal already? At least the journal contains more than than what systemd itself logs about the service start. Might reduce some redundancy and disk I/O if we disable the plain text log file.

However, until we change something, the info here is of course correct.

**Comment by MichaIng** [comment]
An entry on the software overview page is missing: https://github.com/MichaIng/DietPi-Docs/blob/dev/docs/software.md#home-automation

**Comment by Zer0x00** [comment]
> An entry on the software overview page is missing: https://github.com/MichaIng/DietPi-Docs/blob/dev/docs/software.md#home-automation

Fixed in https://github.com/MichaIng/DietPi-Docs/pull/904/commits/4f01a51c06eb9543f70c6c6c811a277aaf4a5744

File: docs/software/home_automation.md (hunk 2 above)
Comment (Zer0x00) [reply docs/software/home_automation.md]: Fixed in https://github.com/MichaIng/DietPi-Docs/pull/904/commits/d7930000fca34c4b85241207c9496fc29ef68ec2

File: docs/software/home_automation.md (hunk 3 above)
Comment (Zer0x00) [reply docs/software/home_automation.md:247]: No, unfortunately the journal has a basic logging like this:
```
root@pi:/opt/homebridge# journalctl -u homebridge
Jul 27 00:37:35 pi systemd[1]: Starting homebridge.service - Homebridge...
Jul 27 00:37:35 pi run-parts[3733]: Ensuring /var/lib/homebridge and /opt/homebridge are owned by homebridge
Jul 27 00:37:35 pi systemd[1]: Started homebridge.service - Homebridge.
Jul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Homebridge Storage Path: /var/lib/homebridgeJul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Homebridge Config Path: /var/lib/homebridge/config.json
Jul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Logging to /var/lib/homebridge/homebridge.log
```

and I don't know a way to disable the logging / move the logging to the journal.

Comment (MichaIng) [reply docs/software/home_automation.md:247]: I cannot find a way to change this either. The official docs do not give a hint either: https://github.com/homebridge/homebridge/wiki/Homebridge-Config-JSON-Explained

I guess it is some library controlling the logs, hence with hidden config keys for it. However, not that important, we can look into it or ask at the HB community at a later time.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**